pkgs/development/libraries/glib/default.nix

```diff
@@ -105,10 +105,10 @@ stdenv.mkDerivation rec {
   ]);
 
   nativeBuildInputs = [
-    meson ninja pkgconfig perl python3 gettext gtk-doc docbook_xsl docbook_xml_dtd_45
+    meson ninja pkgconfig perl python3 gettext-tools gtk-doc docbook_xsl docbook_xml_dtd_45
```

Comment:
Could you make all the alias fixes into a separate commit so it is easier to review?